authentication/oidc.go

```diff
@@ -59,8 +60,8 @@ func NewOIDC(logger log.Logger, configs []TenantOIDCConfig) (http.Handler, map[s
 		}
 
 		r := chi.NewRouter()
-		r.Handle("/login", p.LoginHandler())
-		r.Handle("/callback", p.CallbackHandler())
+		r.Handle("/login", otelhttp.WithRouteTag(prefix+"/login", p.LoginHandler()))
```

Comment:
@kakkoyun I was passing the `prefix` here because these nested routes need to have the correct path in the route tag